setup.py

```diff
@@ -52,6 +52,7 @@ def read_file(filename):
     include_package_data=True,
     install_requires=[
         'Django >= 1.11.2',
+        'django-adminplus >= 0.5',
```

Comment:
For the 'Add areas from a file' admin page. https://github.com/jsocol/django-adminplus